content/d-sgov/d-sgov-klasifikace/d-sgov-klasifikace-model.ttl

```diff
@@ -20,43 +20,589 @@ d-sgov-klasifikace:model a a-popis-dat-pojem:model, owl:Ontology;
+      owl:onProperty z-sgov-pojem:má-vztažený-prvek-2;
+      owl:someValuesFrom d-sgov-klasifikace-pojem:klasifikační-schéma
+    ], [ a owl:Restriction;
+      owl:allValuesFrom d-sgov-klasifikace-pojem:klasifikační-schéma;
+      owl:onProperty z-sgov-pojem:má-vztažený-prvek-2
+    ] .
+
+d-sgov-klasifikace-pojem:nahrazuje-verzi-klasifikace a z-sgov-pojem:typ-vztahu;
+  rdfs:subClassOf d-sgov-klasifikace-pojem:následuje-verzi-klasifikace .
+
+d-sgov-klasifikace-pojem:počet-úrovní-klasifikačního-schématu a z-sgov-pojem:typ-vlastnosti .
+
+d-sgov-klasifikace-pojem:úroveň-klasifikace a z-sgov-pojem:typ-objektu;
+  rdfs:subClassOf [ a owl:Restriction;
+      owl:onProperty z-sgov-pojem:má-vlastnost;
+      owl:someValuesFrom d-sgov-klasifikace-pojem:název-úrovně-klasifikace
+    ], [ a owl:Restriction;
+      owl:allValuesFrom d-sgov-klasifikace-pojem:název-úrovně-klasifikace;
+      owl:onProperty z-sgov-pojem:má-vlastnost
+    ], [ a owl:Restriction;
+      owl:onProperty z-sgov-pojem:má-vlastnost;
+      owl:someValuesFrom d-sgov-klasifikace-pojem:hloubka-úrovně-klasifikace
+    ], [ a owl:Restriction;
+      owl:allValuesFrom d-sgov-klasifikace-pojem:hloubka-úrovně-klasifikace;
+      owl:onProperty z-sgov-pojem:má-vlastnost
+    ] .
+
+d-sgov-klasifikace-pojem:má-úroveň-klasifikace a z-sgov-pojem:typ-vztahu;
+  rdfs:subClassOf [ a owl:Restriction;
+      owl:minQualifiedCardinality "1"^^xsd:nonNegativeInteger;
+      owl:onClass d-sgov-klasifikace-pojem:klasifikační-schéma;
+      owl:onProperty z-sgov-pojem:má-vztažený-prvek-1
+    ], [ a owl:Restriction;
+      owl:maxQualifiedCardinality "1"^^xsd:nonNegativeInteger;
+      owl:onClass d-sgov-klasifikace-pojem:klasifikační-schéma;
+      owl:onProperty z-sgov-pojem:má-vztažený-prvek-1
+    ], [ a owl:Restriction;
+      owl:minQualifiedCardinality "1"^^xsd:nonNegativeInteger;
+      owl:onClass d-sgov-klasifikace-pojem:úroveň-klasifikace;
+      owl:onProperty z-sgov-pojem:má-vztažený-prvek-2
+    ], [ a owl:Restriction;
+      owl:maxQualifiedCardinality "1"^^xsd:nonNegativeInteger;
+      owl:onClass d-sgov-klasifikace-pojem:úroveň-klasifikace;
+      owl:onProperty z-sgov-pojem:má-vztažený-prvek-2
+    ], [ a owl:Restriction;
+      owl:onProperty z-sgov-pojem:má-vztažený-prvek-1;
+      owl:someValuesFrom d-sgov-klasifikace-pojem:klasifikační-schéma
+    ], [ a owl:Restriction;
+      owl:allValuesFrom d-sgov-klasifikace-pojem:klasifikační-schéma;
+      owl:onProperty z-sgov-pojem:má-vztažený-prvek-1
+    ], [ a owl:Restriction;
+      owl:onProperty z-sgov-pojem:má-vztažený-prvek-2;
+      owl:someValuesFrom d-sgov-klasifikace-pojem:úroveň-klasifikace
+    ], [ a owl:Restriction;
+      owl:allValuesFrom d-sgov-klasifikace-pojem:úroveň-klasifikace;
+      owl:onProperty z-sgov-pojem:má-vztažený-prvek-2
+    ] .
+
+d-sgov-klasifikace-pojem:název-úrovně-klasifikace a z-sgov-pojem:typ-vlastnosti .
+
+d-sgov-klasifikace-pojem:hloubka-úrovně-klasifikace a z-sgov-pojem:typ-vlastnosti .
```

Comment:
nemá to být vlastností vztahu "má úroveň klasifikačního schématu" (ale je to mimo OWL compact reprezentaci)

Reply:
Aha, už vidím, že úroveň se nevyskytuje ve více klasifikačních schématech (tedy při vytvoření nového klasifikačního schématu vytváříme nové úrovně). Takže je moje předchozí poznámka bezpředmětná.